src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -1190,3 +1207,177 @@ def run(self):
                 self._future._task_state = FINISHED
 
         return da_rois
+
+def get_tiled_areas(
```

Comment:
The new functions like get_tiled_areas, compute area size, the docsting is missing. Do we need test cases for these two functions?

Reply:
added tests and docstrings 👍